src/Compiler/Checking/MethodCalls.fsi

```diff
@@ -119,7 +119,7 @@ type CallerArgs<'T> =
 /// has been used in F# code
 [<RequireQualifiedAccess>]
 type TypeDirectedConversionUsed =
-    | Yes of (DisplayEnv -> exn)
+    | Yes of (DisplayEnv -> exn) * isDoubleConversion: bool
```

Comment:
The name `DoubleConversion` is problematic - does it mean "int to double" or "there are two conversions".

Also could this better be an integer indicating the number of conversion rules applied?

Reply:
Fair point, something like `isTwoStepConversion` would be a worse sounding but unambiguous name.

Regardless of naming this is a kludge to fix the immediate issue. I would suggest that the PR that fixes #13731 and fully resolves #13437 would introduce richer types like you did in #13063. that would be where we could capture potentially as a linked list (or just an integer indeed) all the conversions done.

Reply:
Yes `isTwoStepConversion` please

Reply:
Renamed